pages/1.13/administering-clusters/package-registry/index.md

```diff
@@ -248,11 +250,11 @@ Mesosphere hosts all its certified packages at https://downloads.mesosphere.com/
       # Make sure the executable works
       ./dcos-registry registry --help
       ```
-      In the rest of the instructions in this page, we assume you have downloaded the subcommand from an attached DC/OS Cluster. If that is not your case, just replace `dcos` with `./dcos-registry` in your instructions.
+      In the rest of the instructions in this page, we assume you have downloaded the subcommand from an attached DC/OS Cluster. If that is not the case, replace `dcos` with `./dcos-registry` in your instructions.
 
 ### Instructions to generate `.dcos` bundle
 
-The `package-registry` cli can be used to bundle your package in to a `.dcos` file that can be used by the `package-registry`. Let us assume that the universe package files are in a directory called `/path/to/package/`. If you list the contents of the this folder, it should have the package definition files:
+The `package-registry` CLI can be used to bundle your package into a `.dcos` file that can be used by the `package-registry`. Let us assume that the {{ model.packageRepo }} package files are in a directory called `/path/to/package/`. If you list the contents of the this folder, it should have the package definition files:
```

Comment:
Remove `the` in `If you list the contents of the this folder,`

Reply:
Rewritten.